src/app/services/components/ExternalServiceDetails.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="stack">
+    <KCard>
+      <template #body>
+        <div
+          class="columns"
+          style="--columns: 3;"
+        >
+          <DefinitionCard>
+            <template #title>
+              {{ t('http.api.property.name') }}
+            </template>
+
+            <template #body>
+              <TextWithCopyButton :text="props.serviceInsight.name">
+                <RouterLink
+                  :to="{
+                    name: 'service-detail-view',
+                    params: {
+                      service: props.serviceInsight.name,
+                      mesh: props.serviceInsight.mesh,
+                    },
+                  }"
+                >
+                  {{ props.serviceInsight.name }}
+                </RouterLink>
+              </TextWithCopyButton>
+            </template>
+          </DefinitionCard>
+
+          <DefinitionCard>
+            <template #title>
+              {{ t('http.api.property.address') }}
+            </template>
+
+            <template #body>
+              {{ props.externalService.networking.address }}
+            </template>
+          </DefinitionCard>
+
+          <DefinitionCard v-if="props.externalService.tags !== null">
+            <template #title>
+              {{ t('http.api.property.tags') }}
+            </template>
+
+            <template #body>
+              <TagList :tags="props.externalService.tags" />
+            </template>
+          </DefinitionCard>
+        </div>
+      </template>
+    </KCard>
+
+    <ResourceCodeBlock
+      id="code-block-service"
+      :resource="props.externalService"
+      :resource-fetcher="fetchExternalService"
+      is-searchable
+      code-max-height="250px"
```

Comment:
Why have a max height btw?

Reply:
I’m pretty sure we added this when the code block used to be next to the details. Now that it’s separate, this should actually be removed. Removing it.